scripts/jenkins/ardana/ansible/deploy-cloud.yml

```diff
@@ -53,6 +53,10 @@
           debug:
             msg: "http://{{ ansible_host }}:9091/"
 
+        - include_role:
```

Comment:
I don't know if you tested this or not, but you should be careful about how you use variables from one role into another, because of this bug: https://github.com/ansible/ansible/issues/21890